.github/workflows/build-multi-os.yml

```diff
@@ -272,6 +272,11 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Patch requirements.txt to use patched version of fbs
+      shell: bash
+      run: |
+        cat requirements.txt | sed 's/fbs==0.8.6/-e git+https:\/\/github.com\/t0mpr1c3\/fbs.git@fbe63505acdf5475e914930ea9de958384a5128c#egg=fbs/' > patched-requirements.txt
```

Comment:
@t0mpr1c3 I would like it better to persistently change the requirements.txt rather than patching it in the workflow, as it somehow "hides" the change.
I'm OK with pointing to the fork at your account, but we can also store it in a repo under the AYAB orga, whatever you prefer.

Reply:
@dl1com I would prefer it be under AYAB so that other people have access. Please could you make an AYAB org fork of fbs, and then I'll raise a PR.

Reply:
> I would like it better to persistently change the requirements.txt rather than patching it in the workflow, as it somehow "hides" the change. 

Yes I agree.

Reply:
> @dl1com I would prefer it be under AYAB so that other people have access. Please could you make an AYAB org fork of fbs, and then I'll raise a PR.

Here we go: https://github.com/AllYarnsAreBeautiful/fbs